fake_robot.py

```diff
@@ -1,10 +1,19 @@
 import time
 
+import gflags
+
 from actions.action import ActionException
 from contextlib import contextmanager
 from parts.load_cell import FakeLoadCellMonitor
 from robot import Robot
 
+FLAGS = gflags.FLAGS
+gflags.DEFINE_boolean("fake_sleep", True, "sleep() in the fake")
+
+def _sleep(secs):
+  if FLAGS.fake_sleep:
```

Comment:
Okay now that I see what it actually does, the comment should probably be "If True, the fake robot will sleep(). Otherwise it will go as fast as possible"